resources/services/glue/workflows.go

```diff
@@ -0,0 +1,256 @@
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("LastRun.Status"),
+			},
+			{
+				Name:          "last_run_workflow_run_id",
+				Description:   "The ID of this workflow run.",
+				Type:          schema.TypeString,
+				Resolver:      schema.PathResolver("LastRun.WorkflowRunId"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "last_run_workflow_run_properties",
+				Description:   "The workflow run properties which were set during the run.",
+				Type:          schema.TypeJSON,
+				Resolver:      schema.PathResolver("LastRun.WorkflowRunProperties"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:        "max_concurrent_runs",
+				Description: "You can use this parameter to prevent unwanted multiple updates to data, to control costs, or in some cases, to prevent exceeding the maximum number of concurrent runs of any of the component jobs",
+				Type:        schema.TypeBigInt,
+			},
+			{
+				Name:        "name",
+				Description: "The name of the workflow.",
+				Type:        schema.TypeString,
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+
+func fetchGlueWorkflows(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().Glue
+	input := glue.ListWorkflowsInput{MaxResults: aws.Int32(25)}
+	for {
+		result, err := svc.ListWorkflows(ctx, &input)
+		if err != nil {
+			return diag.WrapError(err)
+		}
+		for _, name := range result.Workflows {
+			w, err := svc.GetWorkflow(ctx, &glue.GetWorkflowInput{Name: aws.String(name)})
+			if err != nil {
+				if cl.IsNotFoundError(err) {
+					continue
+				}
+				return diag.WrapError(err)
+			}
+			if w.Workflow != nil {
+				res <- *w.Workflow
+			}
+		}
+		if aws.ToString(result.NextToken) == "" {
+			break
+		}
+		input.NextToken = result.NextToken
+	}
+	return nil
+}
+func resolveGlueWorkflowArn(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	cl := meta.(*client.Client)
+	arn := aws.String(workflowARN(cl, aws.ToString(resource.Item.(types.Workflow).Name)))
+	return diag.WrapError(resource.Set(c.Name, arn))
+}
+func resolveGlueWorkflowTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().Glue
+	result, err := svc.GetTags(ctx, &glue.GetTagsInput{
+		ResourceArn: aws.String(workflowARN(cl, aws.ToString(resource.Item.(types.Workflow).Name))),
+	})
+	if err != nil {
+		if cl.IsNotFoundError(err) {
+			return nil
+		}
+		return diag.WrapError(err)
+	}
+	return diag.WrapError(resource.Set(c.Name, result.Tags))
```

Comment:
Use [`client.TagsToMap`](https://github.com/cloudquery/cq-provider-aws/blob/main/client/helpers.go#L415)

Reply:
it's already a map :)